test/resources/case27_showupdateinstatus/case27-create-pod-policy.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: policy.open-cluster-management.io/v1
```

Comment:
Optional: It'd be nice if this didn't use a pod since the pod requires pulling down a container image from the internet to run the test. Perhaps a ConfigMap would be a better option?

Reply:
I changed this but was wondering why we don't want to use pods since we have a lot of tests that use them - is it just to try speed up the test a bit?

Reply:
It's speed but it's also unnecessarily having our tests reaching out over the internet and downloading a decent sized artifact (the container image). It also running an additional container, so it's unnecessary resource utilization.

In general, tests shouldn't reach out to the internet at all unless it's required for the test. This makes them more resilient as all the test's dependencies are local. This particular case, we don't check if the pod is running or not, so even if the container registry was down, the test would still pass, but it's still not best practice based on what I stated earlier.